PMNS_NSIs.h

```diff
@@ -0,0 +1,77 @@
+////////////////////////////////////////////////////////////////////////
+/// \class OscProb::PMNS_NSI
+///
+/// \brief Implementation of oscillations of neutrinos in matter in a
+///        three-neutrino framework with NSI. 
+///
+/// This class expands the PMNS_Fast class including a general
+/// matter potential matrix describing Non-Standard Interactions (NSI).
+///
+/// The matter potential is parametrized by dimensionless quantities
+/// epsilon which quantify the intensity of the NSI with respect to the
+/// standard matter effects from coherent forward scattering with electrons. 
+///
+/// \sa PMNS_Fast
+///
+/// \author coelho\@lal.in2p3.fr
+////////////////////////////////////////////////////////////////////////
+
+#ifndef PMNS_NSIs_H
+#define PMNS_NSIs_H
+
+#include "PMNS_Fast.h"
+
+namespace OscProb {
+  class PMNS_NSIs : public PMNS_Fast {
```

Comment:
Can we call this something more explicit like: `PMNS_ScalarNSI` ?

Reply:
Changed it. Also, I have changed the names of the files to PMNS_ScalarNSI.cxx, and PMNS_ScalarNSI.h